console/console-init/ui/src/graphql-module/queries/Address.ts

```diff
@@ -6,7 +6,10 @@
 import gql from "graphql-tag";
 import { ISortBy } from "@patternfly/react-table";
 import { removeForbiddenChars } from "utils";
-import { getCompplexFilterByPattern, getSimpleFilterByPattern } from "./query";
+import {
+  generateComplexFilterByPattern,
```

Comment:
According to me, simple and complex are a bit confusing and don't give any indication of what the filters are doing. Can we please revisit them?